src/lib/CollisionPrevention/CollisionPrevention.hpp

```diff
@@ -90,6 +95,8 @@ class CollisionPrevention : public ModuleParams
 	matrix::Vector2f _move_constraints_x;
 	matrix::Vector2f _move_constraints_y;
 
+	float _yaw;
```

Comment:
Can't you get this from the attitude subscriber instead of storing it locally? It's not used anywhere else, and you grab the attitude for tilt correction anyway.